### ISSUE TITLE
Sketch v52 - Plugin seems to have stopped working completely

### ISSUE THREAD

**Issue by ghost** [abynim/UserFlows.sketchplugin#141]
Can't add new connections, can't hide or show existing connections. It looks like the layer/art board? the plugin creates has been removed.

**Comment by abynim** [comment]
Yup, I see it. Looks like a lot has changed in 52. I should have it fixed by tomorrow.

Thanks for reporting 👍

**Comment by davidjiangny** [comment]
Seconding this, plugin is not working at all. :(

**Comment by abynim** [comment]
Thanks for reporting folks.
I just pushed an update with fixes for Sketch 52 🎉 
Let me know if I missed anything!

**Comment by ghost** [comment]
All looks well so far! Thanks so much!

**Comment by daCapricorn** [comment]
Failed to generate flow if there is any cross page links. Updated already.

**Comment by abynim** [comment]
Nice catch @daCapricorn. Just pushed a new update (v2.3.1). Let me know if that fixes the issue. Thanks for reporting 👍

**Comment by daCapricorn** [comment]
@abynim work! 👍
